frontend/integration-tests/tests/overview/overview.scenario.ts

```diff
@@ -41,8 +43,10 @@ describe('Visiting Overview page', () => {
       });
 
       it(`shows ${kindModel.id} details sidebar when item is clicked`, async() => {
+        const overviewListItem = overviewView.getProjectOverviewListItem(kindModel, testName);
         await expect(overviewView.detailsSidebar.isPresent()).toBeFalsy();
-        await overviewView.getProjectOverviewListItem(kindModel, testName).click();
+        await browser.wait(until.elementToBeClickable(overviewListItem), BROWSER_TIMEOUT);
```

Comment:
Any reason not to use the default browser timeout? (I think it's actually longer than what you have for `BROWSER_TIMEOUT`)

Reply:
Check other places where we are using `elementToBeClickable` function and on some places we are using 15 seconds, in other we default to 40 seconds.
Thought that 15 seconds should be enough but can bump it to the default 40 :)

Reply:
I don't know the history, but it might have been that the intent was to increase the timeout for these cases...

If we leave it, we should give BROWSER_TIMEOUT a more specific name. That sounds like the global browser timeout and not just for this one case. I'd lean toward just using the default, though.